src/Emulator/Peripherals/Peripherals/Video/STM32LTDC.cs

```diff
@@ -121,7 +124,11 @@ protected override void Repaint()
                     }
                 }
 
-                blender.Blend(localLayerBuffer[0], localLayerBuffer[1], ref buffer, backgroundColor, (byte)layer[0].ConstantAlphaConfigurationRegister.Value, (byte)layer[1].ConstantAlphaConfigurationRegister.Value);
+                blender.Blend(localLayerBuffer[0], localLayerBuffer[1], 
+                    ref buffer, 
+                    backgroundColor, 
+                    (byte)layer[0].ConstantAlphaConfigurationRegister.Value, (layer[1].blendingFactor2.Value == BlendingFactor2.MULTIPLY) ? PixelBlendingMode.MULTIPLY : PixelBlendingMode.NO_MODIFICATION,
+                    (byte)layer[1].ConstantAlphaConfigurationRegister.Value, (layer[1].blendingFactor1.Value == BlendingFactor1.MULTIPLY) ? PixelBlendingMode.MULTIPLY : PixelBlendingMode.NO_MODIFICATION);
```

Comment:
maybe we could split parameters so that we have 1 in line. Otherwise it's a bit confusing